### PR TITLE
faq javafx with ows

### PR DESCRIPTION
**Pull request by janakmulani**

### CODE REVIEW

File: documentation/faq/FAQ.adoc [hunk 1]
```diff
@@ -100,4 +100,117 @@ Note that whitelisting only applies while downloading resources (jars and jnlps)
...
+To be able to run a JavaFX application with OWS using Java 8 requires an installation of Java 8 JVM that includes JavaFX.
+Some of the vendors that have JavaFX as part of their Java 8 JVMs are Oracle, Azul, BellSoft and Amazon. OpenJDK 8 from
+Adopt does not include JavaFX.
+
+Using the JVM Server feature of OWS JVM Manager it can be ensured that a suitable JDK 8 with JavaFX will be installed on
+the machine for OWS to start a JavaFX app. The required JVM from a preferred vendor can be specified in the Jnlp file:
+
+....
+<?xml version="1.0" encoding="utf-8"?>
+<jnlp spec="1.0+" codebase="$$codebase">
```
Comment (AndreasEhret) [inline documentation/faq/FAQ.adoc:118]: use of $$ variable might be confusing here, maybe use some fake url instead?

File: documentation/faq/FAQ.adoc [hunk 2]
```diff
@@ -100,4 +100,117 @@ Note that whitelisting only applies while downloading resources (jars and jnlps)
...
+<information>
+    <title>JavaFX 8 App</title>
+    <vendor>Karakun AG</vendor>
+    <offline-allowed/>
+</information>
+<security>
+    <all-permissions/>
+</security>
+<resources>
+    <java version="1.8*" vendor="zulu" href="http://localhost:8081/jvms/jvms.json"/>
```
Comment (AndreasEhret) [inline documentation/faq/FAQ.adoc:128]: would not use localhost here but a real url

File: documentation/faq/FAQ.adoc [hunk 3]
```diff
@@ -100,4 +100,117 @@ Note that whitelisting only applies while downloading resources (jars and jnlps)
...
+<security>
+    <all-permissions/>
+</security>
+<resources>
+    <java version="11+"/>
+    <jar href="jars/jfxapp.jar"/>
+    <jar href="jars/javafx-controls-11.0.2-win.jar"/>
+    <jar href="jars/javafx-graphics-11.0.2-win.jar"/>
+    <jar href="jars/javafx-base-11.0.2-win.jar"/>
+    <jar href="jars/javafx-fxml-11.0.2-win.jar"/>
```
Comment (AndreasEhret) [inline documentation/faq/FAQ.adoc:187]: maybe you can make a not that this example is the windows flavor and for other OS the jars are accordingly

**Comment by AndreasEhret** [comment]
Please also do not forget to press "PDF" button in intellij asciidoc plugin to esure that the PDF is updated and can checked in

File: documentation/faq/FAQ.adoc (hunk 1 above)
Comment (janakmulani) [reply documentation/faq/FAQ.adoc:118]: This is just a sample. Users knows what to put in codebase.

File: documentation/faq/FAQ.adoc (hunk 2 above)
Comment (janakmulani) [reply documentation/faq/FAQ.adoc:128]: This is just a sample to show you can have a custom JVM server which hosts JVM with FX

File: documentation/faq/FAQ.adoc (hunk 3 above)
Comment (janakmulani) [reply documentation/faq/FAQ.adoc:187]: The note is already there - last bullet before jnlp file

**Pull request merged**